Easy.Logger/AsyncBufferingForwardingAppender.cs

```diff
@@ -91,8 +92,8 @@ protected override void SendBuffer(LoggingEvent[] events)
         /// </summary>
         protected override void OnClose()
         {
-            _idleFlushTimer.Dispose();
-            _sequencer.Shutdown();
+            _idleFlushTimer?.Dispose();
+            _sequencer?.Shutdown();
```

Comment:
`_sequencer` can never be null. Please udno.

Reply:
done